### ISSUE TITLE
[GUIDE] UNinstallation guide for recovery installed BCR 

### ISSUE THREAD

**Issue by drkhemu** [chenxiaolong/BCR#263]
Hello sir,
I have installed BCR via TWRP recovery.
Now I wanted to uninstall it as I want to install via magisk mod.
I tried searching method but couldn't find any.
Please, guide me through uninstall process
Thanks

**Comment by chenxiaolong** [comment]
All you have to do is delete the following paths. You can do it from TWRP or a root file manager.

* `/system/etc/permissions/privapp-permissions-com.chiller3.bcr.xml`
* `/system/priv-app/com.chiller3.bcr/` (entire folder)

**Comment by PritishParihar** [comment]
Thanks for making such a wonderful @chenxiaolong; it served me well for more than 1 year. Recently I have faced the same issue of app keep stopping while going on a call. I've tried above-mentioned steps as suggested above but this file seems in read-only mode. 

 So I've tried deleting with FX File explorer ( with root priv. ) and with TWRP file manager, both seems not working. 

I'm trying to delete and reinstall might fix my issue, also I've tried extracting apk from latest .zip package. 

Any suggestion for me ? 

I'm running Lineage OS 22.2 - Updated on 10 Jan 2025
KernelSU - 12141

Any suggestions would be appreciated :)

**Comment by chenxiaolong** [comment]
Glad you like BCR!

Both Android and TWRP mount the `/system` partition in read-only mode by default. You'll have to remount it as read-write before you can delete files.

In TWRP, I believe there's an option for doing this, but it might be under an advanced settings menu. (I no longer have a device that supports TWRP, so I don't know where exactly it is.)

In Android, you can do it via adb:
```bash
adb shell su -c 'mount -o remount,rw /'
```

**Comment by PritishParihar** [comment]
![Image](https://github.com/user-attachments/assets/4456a95e-9a6a-48fc-a464-2d4dc1746e58)

Facing this issue

**Comment by chenxiaolong** [comment]
Can you copy/paste the output of `cat /proc/self/mountinfo`?

**Comment by PritishParihar** [comment]
> Can you copy/paste the output of `cat /proc/self/mountinfo`?

[output.txt](https://github.com/user-attachments/files/18534993/output.txt)

Here you go :)

**Comment by chenxiaolong** [comment]
Hmm, so it looks like `/system_root` is correct for your device. I'm not sure why it didn't work. Maybe try removing the ending slash you had in the command?

**Comment by PritishParihar** [comment]
> Hmm, so it looks like `/system_root` is correct for your device. I'm not sure why it didn't work. Maybe try removing the ending slash you had in the command?

I have tried removing the ending slash as well but didn't worked for me. 

But I fixed my issues by disablling the app ( might be unnecessary ) since i was not able to remove completely as I have installed with twrp and then clear the data of whole app. After that I rebooted the device and it is working now for me :) But I appreciate for your support and love the app. Btw do lemme know how can I contributed in development of the app or if i can buy you a coffee?
